rotkehlchen/globaldb/updates.py

```diff
@@ -300,74 +305,219 @@ def _parse_full_insert(self, insert_text: str) -> AssetData:
+                        f'Error: {str(e)}',
+                    )
+                return  # fail or succeed continue to next entry
+
+            # else can't resolve. Mark it for the user to resolve.
+            # TODO: When assets refactor is finished, remove the usage of AssetData here
+            local_data = GlobalDBHandler().get_all_asset_data(  # pylint: disable=unsubscriptable-object  # noqa: E501
+                mapping=False,
+                serialized=False,
+                specific_ids=[local_asset.identifier],
+            )[0]
+            self.conflicts.append((local_data, remote_asset_data))
+
     def _apply_single_version_update(
             self,
-            connection: DBConnection,
+            connection: 'DBConnection',
             version: int,
             text: str,
             conflicts: Optional[dict[Asset, Literal['remote', 'local']]],
     ) -> None:
+        """
+        Process each action in the queried data and process it by:
+        1. Try to deserialize it as an asset
+        2. Try to deserialize it as an asset collection
+        3. Try to deserialize it as an asset collection mapping.
+
+        If none of the above rules sucess then the entry is skipped. In the case of assets
+        possible conflicts are handled.
+        """
         lines = text.splitlines()
         for action, full_insert in zip(*[iter(lines)] * 2):
-            if full_insert == '*':
+            if full_insert.replace(' ', '') == '*':
                 full_insert = action
 
+            processed_entry = False
             try:
                 remote_asset_data = self._parse_full_insert(full_insert)
+                processed_entry = True
             except DeserializationError as e:
-                self.msg_aggregator.add_warning(
-                    f'Skipping entry during assets update to v{version} due '
-                    f'to a deserialization error. {str(e)}',
+                log.info(f'Fialed to deserialize {full_insert} as an asset during update to v{version}. Trying asset collection. {str(e)}')  # noqa: E501
+
+            if processed_entry is True:
+                self._handle_asset_update(
+                    connection=connection,
+                    remote_asset_data=remote_asset_data,
+                    conflicts=conflicts,
+                    action=action,
+                    full_insert=full_insert,
+                    version=version,
                 )
                 continue
 
-            local_asset: Optional[Asset] = None
-            with suppress(UnknownAsset):
-                local_asset = Asset(remote_asset_data.identifier).check_existence()
+            # if it wasn't an asset let's check if it is an asset collection
+            try:
+                self._process_asset_collection(
+                    connection=connection,
+                    action=action,
+                    full_insert=full_insert,
```

Comment:
Hmm, I am not sure if `full_insert` is needed for collections (since we probably will never update collections, only add / delete, and then it won't be needed). But not sure.

Reply:
We can rename them for example when an asset changes its name